bootstrapvz/providers/ec2/tasks/packages.py

```diff
@@ -9,9 +10,22 @@ class DefaultPackages(Task):
 	@classmethod
 	def run(cls, info):
 		info.packages.add('file')  # Needed for the init scripts
-		import os.path
+
 		kernel_packages_path = os.path.join(os.path.dirname(__file__), 'packages-kernels.yml')
 		from bootstrapvz.common.tools import config_get
 		kernel_package = config_get(kernel_packages_path, [info.manifest.release.codename,
 		                                                   info.manifest.system['architecture']])
 		info.packages.add(kernel_package)
+
+
+class AddWorkaroundGrowpart(Task):
+	description = 'Adding growpart'
+	phase = phases.system_modification
+
+	@classmethod
+	def run(cls, info):
+		from shutil import copy
+		from . import assets
+		src = os.path.join(assets, 'bin/growpart')
+		dst = os.path.join(info.root, 'usr/bin/growpart')
```

Comment:
I didn't see this before. Aren't we actually overwriting a file here if growpart is installed? Wouldn't it be better to call the script something else and then `sed_i` that name in the init script if we are bootstrapping?